com.stansassets.scene-management/Runtime/Statis/AdditiveScenesLoader.cs

```diff
@@ -127,7 +172,17 @@ public static void Unload(Scene scene, Action unloadCompleted = null)
         /// </summary>
         public static void Unload(string sceneName, Action unloadCompleted = null)
         {
+#if UNITY_EDITOR
             Unload(sceneName, -1, unloadCompleted);
+#else
+            if (BuildConfigurationSettings.Instance.Configuration.IsSceneAddressable(sceneName))
+            {
+                UnloadAddressable(sceneName, unloadCompleted);
```

Comment:
Make sure we do not unload from cache.

Reply:
maybe we will need an additional method for that if we would want to unload addressable scene completely.